eventhub/Module.md

```diff
@@ -1,540 +1,199 @@
-URI to the event hub namespace.
+### Step 2: Create a new connector instance
+Create an `azure_eventhub:ClientEndpointConfiguration` with the extracted shared access key name, shared access key, 
+the resource uri and initialize the connector with it.
 ```ballerina
-    configurable string sasKeyName = ?;
-    configurable string sasKey = ?;
-    configurable string resourceUri = ?;
-
     azure_eventhub:ClientEndpointConfiguration config = {
-        sasKeyName: sasKeyName,
-        sasKey: sasKey,
-        resourceUri: resourceUri 
+        sasKeyName: <SAS_KEY_NAME>,
+        sasKey: <SAS_KEY>,
+        resourceUri: <RESOURCE_URI> 
     };
-    azure_eventhub:Client publisherClient = checkpanic new (config);
-```
-Note:
-You must specify the SAS key name, SAS key and the resource URI when configuring the Azure Event Hub Client connector.
-
-### Step 3: Specify the (Optional) Broker Properties and User Properties
-You can now define the optional broker properties and user properties to be sent with the event using a map.
-```ballerina
-    map<string> brokerProps = {CorrelationId: "34", CorrelationId2: "83"};
-    map<string> userProps = {Alert: "windy", warning: "true"};
-```
 
-### Step 4: Send an event to the Azure Event Hub
-You can now send an event to the Azure event hub by giving the event hub name, and the event hub data with the broker 
-properties and user properties. You can also give a partition key to send events to the same partition with the given 
-partition key name. Here we have sent an event with the string data “eventData” to the event hub named “mytesthub” with 
-the partition key “groupName”.
-```ballerina
-    var sendResult = publisherClient->send("mytesthub", "eventData", userProps, brokerProps, partitionKey = "groupName");
-    if (sendResult is error) {
-            log:printError(sendResult.message());
-    } else {
-            log:printInfo("Successfully Send Event to Event Hub!");
-    }
+    azure_eventhub:Client eventHubClient = check new (config);
 ```
-Note:
-You can specify the event hub path and the event data as parameters of the send method.
-This operation will return a ballerina error if the operation failed.
 
+### Step 3: Invoke connector operation
+1. Now you can use the operations available within the connector. Note that they are in the form of remote operations.
 
-## Entity Management in an Azure Event Hub 
-This is the simplest scenario to manage entities related to azure event hubs. You need to obtain a connection string of 
-the name space of the event hub you want to send events. 
+    Following is an example on how to send an event to the Azure Event Hub using the connector.
 
-### Step 1: Import the Azure Event Hub Ballerina Library
-First, import the ballerinax/azure_eventhub module into the Ballerina project.
-```ballerina
-    import ballerinax/azure_eventhub;
-```
+    Send an event to the Azure Event Hub
```

Comment:
we can use either this or the above line.

Reply:
This is used in the common format.
https://docs.google.com/document/d/1WZg1ix9MpKDTQ_OOm3NVqhQ5L0VGNaUQ1LRmjp-SMO4/edit?usp=sharing